src/selectedtests/app/controllers/project_test_mappings_controller.py

```diff
@@ -47,24 +48,51 @@ def add_project_test_mappings_endpoints(api: Api, mongo: MongoWrapper, evg_api:
         },
     )
 
-    response_body = ns.model(
-        "TestMappingsResponseBody",
-        {"custom": fields.String(description="Message describing the result of the request")},
+    parser = reqparse.RequestParser()
+    parser.add_argument(
+        "changed_files",
+        location="args",
+        help="List of source files to calculate correlated tasks for",
+        required=True,
     )
 
     @ns.route("/<project>/test-mappings")
     @api.param("project", "The evergreen project identifier")
-    class TestMappingsWorkItem(Resource):
-        @ns.response(200, "Success", response_body)
```

Comment:
I didn't think the response_body data (`{"custom": "string"}`) was useful in the API so I removed it:
![Screen Shot 2019-11-15 at 1 26 13 PM](https://user-images.githubusercontent.com/7087489/68968164-ede37700-07af-11ea-88bd-18ad687a22ff.png)